man/systemd.network.xml

```diff
@@ -2050,6 +2082,13 @@ Table=1234</programlisting></para>
           <ulink url="https://tools.ietf.org/html/rfc5227">RFC 5227</ulink>. Defaults to false.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetLabel=</varname></term>
+        <listitem>
+          <para>As in [Address] section.</para>
```

Comment:
"This setting accepts the same arguments as <varname>NetLabel=</varname> in the [Address] section."